homeassistant/components/persistent_notification.py

```diff
@@ -74,7 +76,9 @@ def create_service(call):
 
         hass.states.set(entity_id, message, attr)
 
-    hass.services.register(DOMAIN, SERVICE_CREATE, create_service, {},
-                           SCHEMA_SERVICE_CREATE)
+    descriptions = load_yaml_config_file(
+        os.path.join(os.path.dirname(__file__), 'services.yaml'))
```

Comment:
Is that the only solution? There are many components that need service desriptions that are in root component directory.